### ISSUE TITLE
How setup a Keras Retina-Net model for active learning?

### ISSUE THREAD

**Issue by RafayAK** [microsoft/VoTT#963]
Since the documentation for setting up custom active learning models is pretty sparse. I'm creating this issue to find help in setting up a Keras Retina-Net-like(https://github.com/fizyr/keras-retinanet) model in VoTT.

This model requires some pre and post-processing of the inputs and outputs. My Tensorflow Js model does not do that. I pre-process the image outside the model, pass it in, get some predictions, post-process them, and produce the final output. 

Is there even a way of embedding such a model in VoTT?

Any help will be greatly appreciated.

_BTW I am not really looking for active learning of the Retina-Net Model I just want to use predictions from it to ease the burden of creating a bunch of labels from scratch._
